news/2020-31-12-mappers-report-fall.md

```diff
@@ -0,0 +1,265 @@
+
+For MBC \#8 the restriction was that sliders could not be used, and [Mao](https://osu.ppy.sh/users/2204515)'s entry of [Silentroom - NULCTRL](https://osu.ppy.sh/beatmapsets/1266008#osu/2631203) with its excellent use of circles, **circle size 7**, and screen-usage based patterning won over the judge's hearts. He has now earned enough points in MBC for a tier 2 profile badge, and is all that much closer to attaining the coveted title of elite mapper! Well deserved!
+
+In MBC \#9 the theme was that every object must overlap with another one. With its results mapper [xbopost](https://osu.ppy.sh/users/6842421) came in first place with his entry of [PUP - See You At Your Funeral](https://osu.ppy.sh/beatmapsets/1289138#osu/2676185) which took the theme well to heart and built a unique and interesting map and gameplay experience that you should be sure to check out! That's not all though, as this is xboposts' **third** time coming in first place in MBC and they have now earned the esteemed title of **Elite Mapper**!
+
+In MBC \#10 the restriction was that only sliders could be used, and one mapper coming out of nowhere, [0ppInOsu](https://osu.ppy.sh/users/12551840) managed to take the win! Fully embracing the variable slider velocity and obscure slider-shape style that defines technical mapping, he impressed judges with peak performance in almost all categories.
+For this outstanding performence they were awareded with the tier 1 profile badge. And we hope to see much more from them in the future!
+
+It's certainly well deserved, with each of their entries providing well built and interesting experiences that make excellent use of the challenges MBC provides. They are certainly a mapper to keep up with in the future as well!
+
+The MBC \#10 was also the last one in the year 2020. The contest will take a little break now, but fear not! We are sure that the next MBC won't take long once we are in 2021. So stay tuned and make sure to keep your eyes open for the news post!
+
+### osu! Mapping Olympiad #7: Results (osu!mania)
+
+The results of the osu! Mapping Olympiad are finally here and throwing a big surprise at us. For the first time in history of the Olympiad, two mappers share the 1st place with having the exact same score. Congratulation to [Kawawa](https://osu.ppy.sh/users/4647754) & [Toaph Daddy](https://osu.ppy.sh/users/7616811) being awareded for their outstanding performence with the tier 1 Mapping Olympiad Badge and the **Elite Mapper** user title!
```

Comment:
finally here **,** ~~and~~ throwing a big surprise at us

first time in **the** history

Congratulation**s**

awarded

performance